src/Rules/Node/ForbiddenBlockNodeVisitor.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TwigCsFixer\Rules\Node;
+
+use Twig\Environment;
+use Twig\Node\Node;
+use TwigCsFixer\Rules\AbstractNodeVisitorRule;
+
+final class ForbiddenBlockNodeVisitor extends AbstractNodeVisitorRule
+{
+    /**
+     * @param list<string> $forbidden
+     */
+    public function __construct(
+        private array $forbidden,
+    ) {
+    }
+
+    public function enterNode(Node $node, Environment $env): Node
+    {
+        $blockName = $node->getNodeTag();
+
+        if (null === $blockName) {
+            return $node;
+        }
+
+        if (!\in_array($blockName, $this->forbidden, true)) {
```

Comment:
If you forbid `trans`, I would expect to forbid `TRANS` too.
I'm not sure, but I think twig is not case sensitive, is it ?

We will need to 
- call `strtolower` on forbidden blocks (on the construct maybe)
- call `strtolower` on blockName

Same for others rules.